AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/other-information.model.ts

```diff
@@ -6,10 +6,19 @@ export class OtherInformationModel {
     OtherInformation?: string;
 
     static init(otherInformation: string): OtherInformationModel {
-        let otherInfo: OtherInformationModel = {};
+        let otherInfo: OtherInformationModel = new OtherInformationModel();
         const stringifier = new PipeStringifierService();
         try {
             otherInfo = stringifier.decode(otherInformation);
+            if (!otherInfo.JudgeEmail) {
+                otherInfo.JudgeEmail = null;
+            }
+            if (!otherInfo.JudgePhone) {
+                otherInfo.JudgePhone = null;
+            }
+            if (!otherInfo.OtherInformation) {
+                otherInfo.OtherInformation = null;
+            }
```

Comment:
Probably a nicer way to do this but just ensures that properties are on the object returned previously would return {}